File: common-utils/src/main/java/com/zxn/utils/SimpleTextWatcher.java
package com.zxn.utils;

import android.text.Editable;
import android.text.TextWatcher;

/**
 * 这样可以只重写需要的方法
 * @author user
 *
 */
public class SimpleTextWatcher implements TextWatcher {

	@Override
	public void beforeTextChanged(CharSequence s, int start, int count, int after) {

	}

	@Override
	public void onTextChanged(CharSequence s, int start, int before, int count) {

	}

	@Override
	public void afterTextChanged(Editable s) {

	}

}

File: common-utils/src/main/java/com/zxn/utils/SimpleTextWatcher.kt
package com.zxn.utils

import android.text.Editable
import android.text.TextWatcher

/**
 * 这样可以只重写需要的方法
 * @author user
 */
class SimpleTextWatcher : TextWatcher {
    override fun beforeTextChanged(s: CharSequence, start: Int, count: Int, after: Int) {}
    override fun onTextChanged(s: CharSequence, start: Int, before: Int, count: Int) {}
    override fun afterTextChanged(s: Editable) {}
}